GTG/gtk/editor/editor.py

```diff
@@ -536,14 +614,28 @@ def inserttag_clicked(self, widget):
             self.textview.insert_text("@", itera)
         else:
             self.textview.insert_text(" @", itera)
-        self.textview.grab_focus()
+        # self.textview.grab_focus()
 
     def inserttag(self, widget, tag):
         self.textview.insert_tags([tag])
-        self.textview.grab_focus()
+        # self.textview.grab_focus()
+
+    def remove_tag_edit(self, widget, tagname):
```

Comment:
I don't think it is worth to hide the single call in a special method. Remove it, please.